ext_mongodb.php

```diff
@@ -1410,6 +1469,13 @@ public function serialize() : string
 		] );
 	}
 
+	public function jsonSerialize() : mixed
+	{
+		return [
+			'$date' => (int) $this->milliseconds
```

Comment:
In both strict mode and shell mode, the value should be a string. I believe we want to use an ISO-8601 string for strict mode here.

See: https://docs.mongodb.com/v3.2/reference/mongodb-extended-json/#data_date